### ISSUE TITLE
issue: Minor visual oddness

### ISSUE THREAD

**Issue by tjayrush** [scaffold-eth/scaffold-eth-2#733]
### Is there an existing issue for this?

- [X] I have looked through the [existing issues](https://github.com/scaffold-eth/scaffold-eth-2/issues)

### Which method was used to setup Scaffold-ETH 2 ?

git clone

### Current Behavior

I see two different ways to display an address using 2, 3, or 4 characters in a somewhat random way.

![image](https://github.com/scaffold-eth/scaffold-eth-2/assets/5417918/22ae45b5-a116-4b4d-8e78-a0fdc0708e4b)

### Expected Behavior

Why not be consistent using always four, always three, or always two on either side of the elipses? I know it's anal retentive, but that's my speciality.

### Steps To Reproduce

Look

**Comment by carletex** [comment]
Thanks for this @tjayrush 

Yes! I'm definitely up for having consistency on shortened addresses. I think we should do 4...4 or 3...3.

Thoughts? 

cc @rin-st @technophile-04 @Pabl0cks

**Comment by Pabl0cks** [comment]
> Yes! I'm definitely up for having consistency on shortened addresses. I think we should do 4...4 or 3...3.

Checking most popular dApps 4...4 seems to be the most common pattern, so I would go for it 🙌

**Comment by tjayrush** [comment]
How many places would it have to change, do you think? Can you give me a few pointers to get started?

**Comment by carletex** [comment]
> How many places would it have to change, do you think? Can you give me a few pointers to get started?

I can think of two places:
1. The main one would be in `packages/nextjs/components/scaffold-eth/Address.tsx`
2. And somewhere in `packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton` for the top right wallet.

Thank you! :) Feel free to ask for any guidance.

**Comment by tjayrush** [comment]
Okay. So it's very obvious what to change on this line:

https://github.com/scaffold-eth/scaffold-eth-2/blob/main/packages/nextjs/components/scaffold-eth/Address.tsx#L75

```
  let displayAddress = address?.slice(0, 5) + "..." + address?.slice(-4);
```

(change `.slice(0, 5)` to `.slice(0, 6)`.

But I can't seem to figure out the other change (I'm not very good at Javascript).

I think it's related to this line:

https://github.com/scaffold-eth/scaffold-eth-2/blob/main/packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton/index.tsx#L55

But I can't really figure out where that comes from.

**Comment by technophile-04** [comment]
https://github.com/scaffold-eth/scaffold-eth-2/blob/7c34f62bfccc3ad4be95c1c82dfb680e2b7d4f19/packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton/AddressInfoDropdown.tsx#L50

`displayName` actually comes from Rainbowkit which can be `0x{2 digit}...{4digit}` or ens Name if the address has one 

So we can do here something like : 

```jsx
<span className="ml-2 mr-1">
  {isENS(displayName) ? displayName : address?.slice(0, 5) + "..." + address?.slice(-4)}
</span>
```

We already have [`isENS`](https://github.com/scaffold-eth/scaffold-eth-2/blob/7c34f62bfccc3ad4be95c1c82dfb680e2b7d4f19/packages/nextjs/components/scaffold-eth/Input/utils.ts#L111) util present

**Comment by tjayrush** [comment]
PR created. Thanks for your help. That was pretty easy.

**Comment by technophile-04** [comment]
Closing this fixed at #734
